tests/ui/items_after_test_module.stderr

```diff
@@ -0,0 +1,19 @@
+error: an item was found after the testing module
```

Comment:
Have we considered taking all of the Span between two, and after any testing module, and use that as the lint's Span?

This way the suggestion can be clean when there's many items, imo.

Reply:
The suggestion isn't there anymore

Reply:
Sorry, I think I'm not being clear.

What I'm trying to say is, rather than linting the items one-by-one (which might cause a blow up in the number of clippy warnings/errors in some scenarios), we could emit the warnings in one warning instead.

Also, I think we'd want to lint the macros too. Might be missing something obvious here, but why are we ignoring macros?

Reply:
Sorry, I think I'm not clear enough here.

Rather than linting the items one-by-one (which might cause a blow up in the number of clippy errors/warnings), we can instead collect all the items after the misplaced test module, and lint them as one.

Also, I might be missing something obvious here, but why are we ignoring macros?

Reply:
Ok, I'm pretty dumb.
The macros were being ignored because I thought that the user couldn't move them, as the place of macro declaration matters a lot to the usage of it. But thinking a little bit more, it doesn't the matter, as everything after the testing module should be moved in pack.

I'm going to fix this now.

Reply:
Fixed, for real now

Reply:
Not a bit of worry. Thanks for incorporating the suggestion above :)